code.user.js

```diff
@@ -1,38 +1,42 @@
 // ==UserScript==
-// @name        Steam Economy Enhancer
-// @icon        https://upload.wikimedia.org/wikipedia/commons/8/83/Steam_icon_logo.svg
-// @namespace   https://github.com/Nuklon
-// @author      Nuklon
-// @license     MIT
-// @version     6.9.3
-// @description Enhances the Steam Inventory and Steam Market.
-// @include     *://steamcommunity.com/id/*/inventory*
-// @include     *://steamcommunity.com/profiles/*/inventory*
-// @include     *://steamcommunity.com/market*
-// @include     *://steamcommunity.com/tradeoffer*
-// @require     https://code.jquery.com/jquery-3.3.1.min.js
-// @require     https://code.jquery.com/ui/1.12.1/jquery-ui.min.js
-// @require     https://raw.githubusercontent.com/kapetan/jquery-observe/ca67b735bb3ae8d678d1843384ebbe7c02466c61/jquery-observe.js
-// @require     https://cdnjs.cloudflare.com/ajax/libs/paginationjs/2.1.2/pagination.min.js
-// @require     https://cdnjs.cloudflare.com/ajax/libs/async/2.6.0/async.js
-// @require     https://cdnjs.cloudflare.com/ajax/libs/localforage/1.7.1/localforage.min.js
-// @require     https://moment.github.io/luxon/global/luxon.min.js
-// @require     https://cdnjs.cloudflare.com/ajax/libs/list.js/1.5.0/list.js
-// @require     https://raw.githubusercontent.com/rmariuzzo/checkboxes.js/91bec667e9172ceb063df1ecb7505e8ed0bae9ba/src/jquery.checkboxes.js
-// @grant       unsafeWindow
-// @homepageURL https://github.com/Nuklon/Steam-Economy-Enhancer
-// @supportURL  https://github.com/Nuklon/Steam-Economy-Enhancer/issues
-// @downloadURL https://raw.githubusercontent.com/Nuklon/Steam-Economy-Enhancer/master/code.user.js
-// @updateURL   https://raw.githubusercontent.com/Nuklon/Steam-Economy-Enhancer/master/code.user.js
+// @name         Steam Economy Enhancer
+// @icon         https://upload.wikimedia.org/wikipedia/commons/8/83/Steam_icon_logo.svg
+// @namespace    https://github.com/Nuklon
+// @author       Nuklon
+// @license      MIT
+// @version      6.9.3
+// @description  Enhances the Steam Inventory and Steam Market.
+// @match        https://steamcommunity.com/id/*/inventory*
```

Comment:
I changed \@include to \@match as the userscripts lint suggested that for manifest v3.